src/xr_interface_openvr.cpp

```diff
@@ -351,18 +351,24 @@ void XRInterfaceOpenVR::_commit_views(const RID &p_render_target, const Rect2 &p
 		}
 	}
 
+	texture_rid = get_render_target_texture(p_render_target);
+};
+
+void XRInterfaceOpenVR::_end_frame() {
+	if (!ovr->is_initialised()) {
```

Comment:
Should probably leave a comment here about why we do this or reference back to the docs for _end_frame()

Reply:
Good idea, done :)